release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -144,6 +144,77 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-storage"]
 === Storage
 
+==== CSI volume health (Technology Preview)
+Container Storage Interface (CSI) volume health is currently enabled, and supported as a Technology Preview feature. However, {product-title} currently does not ship any CSI drivers that support this feature.
+
+==== CSI driver volume detach with non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
```

Comment:
`you can manually add an out-of-service taint`

Does this needs to be done before the shutdown? I assume so; but it isn't 100% clear to me.

Reply:
Actually not. I'll correct the wording.